### DISCUSSION TITLE
Turbopack Inline SVG support

### DISCUSSION THREAD

**Discussion by agusterodin** [vercel/turborepo#2401] [Ideas]
Hey! Absolutely love what you are doing with Turbopack and the web ecosystem as a whole.

Importing SVGs as inline React components is an extremely common use case. Nearly all projects I have touched use the SVGR webpack plugin to achieve this.

Is there a possibility of future first-class support for this kind of thing with Turbopack?

**Comment by nathanhammond** [comment]
Stay tuned.
